### ISSUE TITLE
Add licensing and/or advise on use of grants

### ISSUE THREAD

**Issue by ethanwhite** [weecology/ogrants#113]
The site doesn't currently include any information on copyright/licensing and no discussion/advice of the norms of use (to the extent that they exist) for publicly posted grant proposals. Thanks to @AnneCarpenter for bringing up this important import point.

Things to consider:

1. The license (or lack their of) is set by the person posting the individual grants. So, for example, a lot of my proposals have CC-BY licenses on them but many of the other proposals are simply posted publicly on the internet with no associated license.
2. There's a question of culturally acceptable use vs. license. So, while a CC0 public domain declaration (which I believe at least a couple of the proposals may be posted under) would allow the copying and pasting of text into something else even without attribution from a legal perspective, culturally defined citation and plagiarism standards would deem this unacceptable.

So, how should we handle this at Ogrants? I guess I see a couple of options:

1. Provide more grant-level information. We could add a copyright/license field to each grant and present that information to the user. This would communicate legal rights and author intent to the degree that it is specified by the license.
2. Provide general guidance on the use of the linked proposals. We could add a page that discusses what we perceive as the existing norms for the use of this material and provide guidance on avoiding issues when using others grants during the grant writing process (e.g., copying structure is generally fine, copying actual language is not).

My initial take is that both of these would be valuable improvements to the site. I'm interested in hearing others thoughts. @AnneCarpenter had some good thoughts that she brought up over email, which I'll let her add here if she would like.

**Comment by AnneCarpenter** [comment]
Thanks for kicking off this discussion!

I think it would be great to provide guidance on what is academically acceptable use of others' example text, because many early-career writers are not aware of the norms about plagiarism and attribution. I wonder if there are any public descriptions of this we could point to (with attribution of course :D ) instead of reinventing the wheel?

Regarding actual licenses, I suspect a minimalist approach would do: there could be a site-wide announcement that all materials are CC-BY unless otherwise noted therein. That allows researchers to add a more lenient or stringent license if they like, but most people don't need to think about it. I think offering different choices at the grant level might inadvertently reinforce the  
idea that it is okay to follow legal norms rather than academic/cultural norms.

**Comment by dhimmel** [comment]
> I think it would be great to provide guidance on what is academically acceptable use of others' example text, because many early-career writers are not aware of the norms about plagiarism and attribution.

Good idea.

> We could add a copyright/license field to each grant and present that information to the user

It would be helpful to know the license of the grant documents from the ogrants webpage. But I am worried about a situation where ogrants claims a work is released under a certain license when it actually is not.

Since ogrants doesn't actually host the grants and often ogrants are uploaded by someone other than the copyright holder of the grant, I don't think ogrants is the right place to define a grant's license. If a grant already has a license applied on the site that hosts it, then its okay for ogrants to reflect that license.

But what if ogrant curators mistakenly note the license or the upstream license changes... then ogrants could make some users think a grant was available under a license it isn't.

> there could be a site-wide announcement that all materials are CC-BY unless otherwise noted therein

I think this is risky. A public license should always be an opt-in kind of thing. For example, what about all of the existing grants, many of which are probably all rights reserved.

**Comment by nuest** [comment]
Related to licensing: It would be great if the "data" published on Open Grants would be clearly licensed as well, e.g. PDDL ? Or CC0? Should be covered by a statement in the README and in the footer of each page. The MIT license used for the repository is not suitable for a dataset.

**Comment by dhimmel** [comment]
> It would be great if the "data" published on Open Grants would be clearly licensed as well

Yeah the releasing the opengrants dataset under CC0 would be good. You could continue having the MIT license apply to everything in the repo, and then specify specific data files to also license as CC0.

**Comment by ethanwhite** [comment]
Yep, good call. Can someone open this as a separate issue when they get a chance (or even better a PR :smile: )

**Comment by trashbirdecology** [comment]
All proposal materials are linked externally and most (?) have specified licensing at that location. Would it suffice to suggest in the README that readers determine licensing at those external sources and/or reach out to the author(s)?

**Comment by ha0ye** [comment]
Agree with the suggestion for a short primer on guidance for usage and link to external resources on copyright/licensing/plagiarism is useful. e.g. https://guides.uflib.ufl.edu/copyright/plagiarism

IMO, the key points are:
* Lack of a license on linked grant materials does not preclude citation of it as a scholarly object. (just like citing a journal article with copyright but all rights reserved)
* Licensing is determined by the copyright holder at the source, which will usually be the author(s) unless otherwise transferred.
* A license, such as CC-BY, specifies permissions for reproducing part or all of the work.
* A license that permits reproduction, does NOT permit improper or absent documentation of the source material. (i.e. the distinction between legal use vs. culturally acceptable use)
